### ISSUE TITLE
Request initialize failed with message: ENOENT: no such file or directory, open 'package.json'

### ISSUE THREAD

**Issue by mrmlnc** [SamVerschueren/vscode-linter-xo#24]
Hello, @SamVerschueren,

After the last update I started getting error if workspace have no `package.json` file:

```
Request initialize failed with message: ENOENT: no such file or directory, open 'c:\projects\tests\xo-error\package.json'
```

![code_2016-12-24_14-03-34](https://cloud.githubusercontent.com/assets/7034281/21466644/6c2ff8f4-c9e2-11e6-9be3-d62055f24ede.png)

It seems to me that it's related to https://github.com/SamVerschueren/vscode-linter-xo/blob/master/xo-server/src/package.ts#L11

## Steps to reproduce
 
  1. Create empty directory with `test.js` file
  2. Run `code .`
  3. Look at error

## Techs

  * Windows 10
  * VS Code 1.8.1

**Comment by SamVerschueren** [comment]
Thanks for the report. This is correct, I changed to use load-json-file but didn't implement error handling correctly. Don't have much time now to fix it so a PR is welcome. Or wait until I find some time :).

**Comment by mrmlnc** [comment]
IMO, in this case we shouldn't show any errors — just silently return false for this workspace. I'll send PR soon :)

**Comment by SamVerschueren** [comment]
Should be fixed in `2.1.2`. Thanks for reporting!
